guides/getting-started.md

```diff
@@ -134,6 +134,8 @@ For more information on using Material Icons, check out the
 
 Note that `mat-icon` supports any font or svg icons; using Material Icons is one of many options.
 
+For more information on using `mat-icon` component for both font and svg icons, check out the 
```

Comment:
* "on using `mat-icon` component" -> "on using the `mat-icon` component"
* svg -> SVG